API/Models/Camera.cs

```diff
@@ -5,7 +5,8 @@ namespace API.Models
 {
     public enum Side
     {
-        Main, Selfie
+        Main,
```

Comment:
Could this enum have its values set to same string for ex. Main = "Main", Selife = "Selfie". Since I assumed that's what backend would give because now it probably gives back integers

Reply:
Enums by default in C# only have int values. It would be possible to move this into database and then join by query. Not in the scope of this MR, though.

Reply:
Yikes didn't know C# was so shit on this one